src/Beberlei/Metrics/Collector/Librato.php

```diff
@@ -71,15 +97,20 @@ public function measure($variable, $value)
         );
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function flush()
     {
         if (!$this->data['gauges'] && !$this->data['counters']) {
             return;
         }
 
+        $authToken = base64_encode($this->username.':'.$this->password);
```

Comment:
And Now I fail to see why you change this. 
You never re-use the `$authToken`; So everything on the same line is better.